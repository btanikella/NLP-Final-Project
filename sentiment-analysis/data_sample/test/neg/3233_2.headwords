going
butchered
lifeless
gives
... okay maybe C.H.O.M.P.S. `` If
avoid
read
thriller
