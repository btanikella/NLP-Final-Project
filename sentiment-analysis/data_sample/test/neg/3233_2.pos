NNP NNP VBZ RB VBG TO VB VBN IN CD IN DT JJ NNS IN PRP$ NN  CC IN JJS IN NNS IN  JJ TO VB   PRP VBD RBR IN PRP$ NN : JJ JJ NNP POS NN  NNPS  VBD CD IN PRP$ JJR NNS  CC RB RB PRP$ JJS TO NN
RB  DT JJ NN IN DT JJ NN  DT JJ NN CC DT JJ NN VBN JJ VBZ VBN RB
DT NN VBZ RB JJ  PRP MD VB PRP VBP VBG DT NN NN
RB DT NN VBZ DT JJ NN  CC IN PRP VBP TO VB DT JJ NN IN DT NN  PRP MD VB JJR IN VBG  DT JJ NN    NNP  CC RB  NNP
: JJ  RB RB  NNP  IN
PRP VBP VBN DT NN  VB DT NN IN DT NNS
IN PRP VBP RB VB DT NN  VBD PRP CC VB DT NN
PRP MD VB PRP JJ RB JJR NN IN DT NNP NNP NN VBZ DT JJ NN   NNP
