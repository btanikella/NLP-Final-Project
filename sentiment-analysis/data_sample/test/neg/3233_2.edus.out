Sentence #1 (60 tokens):
Corey Haim is never going to be known as one of the great actors of his time, but at least in movies like "Licensed To Drive", he was more in his element... lowbrow humor.Dean Koontz's book "Watchers" was one of his earlier works, and still probably his finest to date.
[Text=Corey CharacterOffsetBegin=4 CharacterOffsetEnd=9 PartOfSpeech=NNP] [Text=Haim CharacterOffsetBegin=10 CharacterOffsetEnd=14 PartOfSpeech=NNP] [Text=is CharacterOffsetBegin=15 CharacterOffsetEnd=17 PartOfSpeech=VBZ] [Text=never CharacterOffsetBegin=18 CharacterOffsetEnd=23 PartOfSpeech=RB] [Text=going CharacterOffsetBegin=24 CharacterOffsetEnd=29 PartOfSpeech=VBG] [Text=to CharacterOffsetBegin=30 CharacterOffsetEnd=32 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=33 CharacterOffsetEnd=35 PartOfSpeech=VB] [Text=known CharacterOffsetBegin=36 CharacterOffsetEnd=41 PartOfSpeech=VBN] [Text=as CharacterOffsetBegin=42 CharacterOffsetEnd=44 PartOfSpeech=IN] [Text=one CharacterOffsetBegin=45 CharacterOffsetEnd=48 PartOfSpeech=CD] [Text=of CharacterOffsetBegin=49 CharacterOffsetEnd=51 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=52 CharacterOffsetEnd=55 PartOfSpeech=DT] [Text=great CharacterOffsetBegin=56 CharacterOffsetEnd=61 PartOfSpeech=JJ] [Text=actors CharacterOffsetBegin=62 CharacterOffsetEnd=68 PartOfSpeech=NNS] [Text=of CharacterOffsetBegin=69 CharacterOffsetEnd=71 PartOfSpeech=IN] [Text=his CharacterOffsetBegin=72 CharacterOffsetEnd=75 PartOfSpeech=PRP$] [Text=time CharacterOffsetBegin=76 CharacterOffsetEnd=80 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=80 CharacterOffsetEnd=81 PartOfSpeech=,] [Text=but CharacterOffsetBegin=82 CharacterOffsetEnd=85 PartOfSpeech=CC] [Text=at CharacterOffsetBegin=86 CharacterOffsetEnd=88 PartOfSpeech=IN] [Text=least CharacterOffsetBegin=89 CharacterOffsetEnd=94 PartOfSpeech=JJS] [Text=in CharacterOffsetBegin=95 CharacterOffsetEnd=97 PartOfSpeech=IN] [Text=movies CharacterOffsetBegin=98 CharacterOffsetEnd=104 PartOfSpeech=NNS] [Text=like CharacterOffsetBegin=105 CharacterOffsetEnd=109 PartOfSpeech=IN] [Text=`` CharacterOffsetBegin=110 CharacterOffsetEnd=111 PartOfSpeech=``] [Text=Licensed CharacterOffsetBegin=111 CharacterOffsetEnd=119 PartOfSpeech=JJ] [Text=To CharacterOffsetBegin=120 CharacterOffsetEnd=122 PartOfSpeech=TO] [Text=Drive CharacterOffsetBegin=123 CharacterOffsetEnd=128 PartOfSpeech=VB] [Text='' CharacterOffsetBegin=128 CharacterOffsetEnd=129 PartOfSpeech=''] [Text=, CharacterOffsetBegin=129 CharacterOffsetEnd=130 PartOfSpeech=,] [Text=he CharacterOffsetBegin=131 CharacterOffsetEnd=133 PartOfSpeech=PRP] [Text=was CharacterOffsetBegin=134 CharacterOffsetEnd=137 PartOfSpeech=VBD] [Text=more CharacterOffsetBegin=138 CharacterOffsetEnd=142 PartOfSpeech=RBR] [Text=in CharacterOffsetBegin=143 CharacterOffsetEnd=145 PartOfSpeech=IN] [Text=his CharacterOffsetBegin=146 CharacterOffsetEnd=149 PartOfSpeech=PRP$] [Text=element CharacterOffsetBegin=150 CharacterOffsetEnd=157 PartOfSpeech=NN] [Text=... CharacterOffsetBegin=157 CharacterOffsetEnd=160 PartOfSpeech=:] [Text=lowbrow CharacterOffsetBegin=161 CharacterOffsetEnd=168 PartOfSpeech=JJ] [Text=humor.Dean CharacterOffsetBegin=169 CharacterOffsetEnd=179 PartOfSpeech=JJ] [Text=Koontz CharacterOffsetBegin=180 CharacterOffsetEnd=186 PartOfSpeech=NNP] [Text='s CharacterOffsetBegin=186 CharacterOffsetEnd=188 PartOfSpeech=POS] [Text=book CharacterOffsetBegin=189 CharacterOffsetEnd=193 PartOfSpeech=NN] [Text=`` CharacterOffsetBegin=194 CharacterOffsetEnd=195 PartOfSpeech=``] [Text=Watchers CharacterOffsetBegin=195 CharacterOffsetEnd=203 PartOfSpeech=NNPS] [Text='' CharacterOffsetBegin=203 CharacterOffsetEnd=204 PartOfSpeech=''] [Text=was CharacterOffsetBegin=205 CharacterOffsetEnd=208 PartOfSpeech=VBD] [Text=one CharacterOffsetBegin=209 CharacterOffsetEnd=212 PartOfSpeech=CD] [Text=of CharacterOffsetBegin=213 CharacterOffsetEnd=215 PartOfSpeech=IN] [Text=his CharacterOffsetBegin=216 CharacterOffsetEnd=219 PartOfSpeech=PRP$] [Text=earlier CharacterOffsetBegin=220 CharacterOffsetEnd=227 PartOfSpeech=JJR] [Text=works CharacterOffsetBegin=228 CharacterOffsetEnd=233 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=233 CharacterOffsetEnd=234 PartOfSpeech=,] [Text=and CharacterOffsetBegin=235 CharacterOffsetEnd=238 PartOfSpeech=CC] [Text=still CharacterOffsetBegin=239 CharacterOffsetEnd=244 PartOfSpeech=RB] [Text=probably CharacterOffsetBegin=245 CharacterOffsetEnd=253 PartOfSpeech=RB] [Text=his CharacterOffsetBegin=254 CharacterOffsetEnd=257 PartOfSpeech=PRP$] [Text=finest CharacterOffsetBegin=258 CharacterOffsetEnd=264 PartOfSpeech=JJS] [Text=to CharacterOffsetBegin=265 CharacterOffsetEnd=267 PartOfSpeech=TO] [Text=date CharacterOffsetBegin=268 CharacterOffsetEnd=272 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=272 CharacterOffsetEnd=273 PartOfSpeech=.] 
root(ROOT-0, going-5)
nn(Haim-2, Corey-1)
nsubj(going-5, Haim-2)
aux(going-5, is-3)
neg(going-5, never-4)
aux(known-8, to-6)
auxpass(known-8, be-7)
xcomp(going-5, known-8)
prep(known-8, as-9)
pobj(as-9, one-10)
prep(one-10, of-11)
det(actors-14, the-12)
amod(actors-14, great-13)
pobj(of-11, actors-14)
prep(actors-14, of-15)
poss(time-17, his-16)
pobj(of-15, time-17)
punct(going-5, ,-18)
cc(going-5, but-19)
advmod(in-22, at-20)
pobj(at-20, least-21)
conj(going-5, in-22)
pobj(in-22, movies-23)
prep(movies-23, like-24)
punct(like-24, ``-25)
conj(like-24, Licensed-26)
aux(Drive-28, To-27)
xcomp(Licensed-26, Drive-28)
punct(Drive-28, ''-29)
punct(like-24, ,-30)
nsubj(more-33, he-31)
cop(more-33, was-32)
conj(like-24, more-33)
prep(more-33, in-34)
poss(element-36, his-35)
pobj(in-34, element-36)
punct(like-24, ...-37)
nsubj(one-47, lowbrow-38)
amod(Koontz-40, humor.Dean-39)
poss(book-42, Koontz-40)
possessive(Koontz-40, 's-41)
dep(lowbrow-38, book-42)
punct(Watchers-44, ``-43)
dep(book-42, Watchers-44)
punct(Watchers-44, ''-45)
cop(one-47, was-46)
dep(like-24, one-47)
prep(one-47, of-48)
poss(works-51, his-49)
amod(works-51, earlier-50)
pobj(of-48, works-51)
punct(one-47, ,-52)
cc(one-47, and-53)
advmod(one-47, still-54)
advmod(finest-57, probably-55)
poss(finest-57, his-56)
dep(one-47, finest-57)
prep(finest-57, to-58)
pobj(to-58, date-59)
punct(going-5, .-60)

Sentence #2 (23 tokens):
Sadly, this magnificent tale of a brilliant dog, a deranged mutant and a genetic experiment gone wrong is butchered horribly.
[Text=Sadly CharacterOffsetBegin=278 CharacterOffsetEnd=283 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=283 CharacterOffsetEnd=284 PartOfSpeech=,] [Text=this CharacterOffsetBegin=285 CharacterOffsetEnd=289 PartOfSpeech=DT] [Text=magnificent CharacterOffsetBegin=290 CharacterOffsetEnd=301 PartOfSpeech=JJ] [Text=tale CharacterOffsetBegin=302 CharacterOffsetEnd=306 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=307 CharacterOffsetEnd=309 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=310 CharacterOffsetEnd=311 PartOfSpeech=DT] [Text=brilliant CharacterOffsetBegin=312 CharacterOffsetEnd=321 PartOfSpeech=JJ] [Text=dog CharacterOffsetBegin=322 CharacterOffsetEnd=325 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=325 CharacterOffsetEnd=326 PartOfSpeech=,] [Text=a CharacterOffsetBegin=327 CharacterOffsetEnd=328 PartOfSpeech=DT] [Text=deranged CharacterOffsetBegin=329 CharacterOffsetEnd=337 PartOfSpeech=JJ] [Text=mutant CharacterOffsetBegin=338 CharacterOffsetEnd=344 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=345 CharacterOffsetEnd=348 PartOfSpeech=CC] [Text=a CharacterOffsetBegin=349 CharacterOffsetEnd=350 PartOfSpeech=DT] [Text=genetic CharacterOffsetBegin=351 CharacterOffsetEnd=358 PartOfSpeech=JJ] [Text=experiment CharacterOffsetBegin=359 CharacterOffsetEnd=369 PartOfSpeech=NN] [Text=gone CharacterOffsetBegin=370 CharacterOffsetEnd=374 PartOfSpeech=VBN] [Text=wrong CharacterOffsetBegin=375 CharacterOffsetEnd=380 PartOfSpeech=JJ] [Text=is CharacterOffsetBegin=381 CharacterOffsetEnd=383 PartOfSpeech=VBZ] [Text=butchered CharacterOffsetBegin=384 CharacterOffsetEnd=393 PartOfSpeech=VBN] [Text=horribly CharacterOffsetBegin=394 CharacterOffsetEnd=402 PartOfSpeech=RB] [Text=. CharacterOffsetBegin=402 CharacterOffsetEnd=403 PartOfSpeech=.] 
root(ROOT-0, butchered-21)
advmod(butchered-21, Sadly-1)
punct(butchered-21, ,-2)
det(tale-5, this-3)
amod(tale-5, magnificent-4)
nsubjpass(butchered-21, tale-5)
prep(tale-5, of-6)
det(dog-9, a-7)
amod(dog-9, brilliant-8)
pobj(of-6, dog-9)
punct(dog-9, ,-10)
det(mutant-13, a-11)
amod(mutant-13, deranged-12)
conj(dog-9, mutant-13)
cc(dog-9, and-14)
det(experiment-17, a-15)
amod(experiment-17, genetic-16)
conj(dog-9, experiment-17)
partmod(experiment-17, gone-18)
acomp(gone-18, wrong-19)
auxpass(butchered-21, is-20)
advmod(butchered-21, horribly-22)
punct(butchered-21, .-23)

Sentence #3 (16 tokens):
The acting is so lifeless, you might think you're watching a zombie movie.
[Text=The CharacterOffsetBegin=408 CharacterOffsetEnd=411 PartOfSpeech=DT] [Text=acting CharacterOffsetBegin=412 CharacterOffsetEnd=418 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=419 CharacterOffsetEnd=421 PartOfSpeech=VBZ] [Text=so CharacterOffsetBegin=422 CharacterOffsetEnd=424 PartOfSpeech=RB] [Text=lifeless CharacterOffsetBegin=425 CharacterOffsetEnd=433 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=433 CharacterOffsetEnd=434 PartOfSpeech=,] [Text=you CharacterOffsetBegin=435 CharacterOffsetEnd=438 PartOfSpeech=PRP] [Text=might CharacterOffsetBegin=439 CharacterOffsetEnd=444 PartOfSpeech=MD] [Text=think CharacterOffsetBegin=445 CharacterOffsetEnd=450 PartOfSpeech=VB] [Text=you CharacterOffsetBegin=451 CharacterOffsetEnd=454 PartOfSpeech=PRP] [Text='re CharacterOffsetBegin=454 CharacterOffsetEnd=457 PartOfSpeech=VBP] [Text=watching CharacterOffsetBegin=458 CharacterOffsetEnd=466 PartOfSpeech=VBG] [Text=a CharacterOffsetBegin=467 CharacterOffsetEnd=468 PartOfSpeech=DT] [Text=zombie CharacterOffsetBegin=469 CharacterOffsetEnd=475 PartOfSpeech=NN] [Text=movie CharacterOffsetBegin=476 CharacterOffsetEnd=481 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=481 CharacterOffsetEnd=482 PartOfSpeech=.] 
root(ROOT-0, lifeless-5)
det(acting-2, The-1)
nsubj(lifeless-5, acting-2)
cop(lifeless-5, is-3)
advmod(lifeless-5, so-4)
punct(lifeless-5, ,-6)
nsubj(think-9, you-7)
aux(think-9, might-8)
ccomp(lifeless-5, think-9)
nsubj(watching-12, you-10)
aux(watching-12, 're-11)
ccomp(think-9, watching-12)
det(movie-15, a-13)
nn(movie-15, zombie-14)
dobj(watching-12, movie-15)
punct(lifeless-5, .-16)

Sentence #4 (63 tokens):
Only the dog gives a respectable performance, and if you want to see a decent movie about a dog, you'd be better of watching "The Incredible Journey", "Cujo" or even "C.H.O.M.P.S.
    "...okay, maybe not "C.H.O.M.P.S.
    "If you've read the book, avoid this movie at all costs.
[Text=Only CharacterOffsetBegin=487 CharacterOffsetEnd=491 PartOfSpeech=RB] [Text=the CharacterOffsetBegin=492 CharacterOffsetEnd=495 PartOfSpeech=DT] [Text=dog CharacterOffsetBegin=496 CharacterOffsetEnd=499 PartOfSpeech=NN] [Text=gives CharacterOffsetBegin=500 CharacterOffsetEnd=505 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=506 CharacterOffsetEnd=507 PartOfSpeech=DT] [Text=respectable CharacterOffsetBegin=508 CharacterOffsetEnd=519 PartOfSpeech=JJ] [Text=performance CharacterOffsetBegin=520 CharacterOffsetEnd=531 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=531 CharacterOffsetEnd=532 PartOfSpeech=,] [Text=and CharacterOffsetBegin=533 CharacterOffsetEnd=536 PartOfSpeech=CC] [Text=if CharacterOffsetBegin=537 CharacterOffsetEnd=539 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=540 CharacterOffsetEnd=543 PartOfSpeech=PRP] [Text=want CharacterOffsetBegin=544 CharacterOffsetEnd=548 PartOfSpeech=VBP] [Text=to CharacterOffsetBegin=549 CharacterOffsetEnd=551 PartOfSpeech=TO] [Text=see CharacterOffsetBegin=552 CharacterOffsetEnd=555 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=556 CharacterOffsetEnd=557 PartOfSpeech=DT] [Text=decent CharacterOffsetBegin=558 CharacterOffsetEnd=564 PartOfSpeech=JJ] [Text=movie CharacterOffsetBegin=565 CharacterOffsetEnd=570 PartOfSpeech=NN] [Text=about CharacterOffsetBegin=571 CharacterOffsetEnd=576 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=577 CharacterOffsetEnd=578 PartOfSpeech=DT] [Text=dog CharacterOffsetBegin=579 CharacterOffsetEnd=582 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=582 CharacterOffsetEnd=583 PartOfSpeech=,] [Text=you CharacterOffsetBegin=584 CharacterOffsetEnd=587 PartOfSpeech=PRP] [Text='d CharacterOffsetBegin=587 CharacterOffsetEnd=589 PartOfSpeech=MD] [Text=be CharacterOffsetBegin=590 CharacterOffsetEnd=592 PartOfSpeech=VB] [Text=better CharacterOffsetBegin=593 CharacterOffsetEnd=599 PartOfSpeech=JJR] [Text=of CharacterOffsetBegin=600 CharacterOffsetEnd=602 PartOfSpeech=IN] [Text=watching CharacterOffsetBegin=603 CharacterOffsetEnd=611 PartOfSpeech=VBG] [Text=`` CharacterOffsetBegin=612 CharacterOffsetEnd=613 PartOfSpeech=``] [Text=The CharacterOffsetBegin=613 CharacterOffsetEnd=616 PartOfSpeech=DT] [Text=Incredible CharacterOffsetBegin=617 CharacterOffsetEnd=627 PartOfSpeech=JJ] [Text=Journey CharacterOffsetBegin=628 CharacterOffsetEnd=635 PartOfSpeech=NN] [Text='' CharacterOffsetBegin=635 CharacterOffsetEnd=636 PartOfSpeech=''] [Text=, CharacterOffsetBegin=636 CharacterOffsetEnd=637 PartOfSpeech=,] [Text=`` CharacterOffsetBegin=638 CharacterOffsetEnd=639 PartOfSpeech=``] [Text=Cujo CharacterOffsetBegin=639 CharacterOffsetEnd=643 PartOfSpeech=NNP] [Text='' CharacterOffsetBegin=643 CharacterOffsetEnd=644 PartOfSpeech=''] [Text=or CharacterOffsetBegin=645 CharacterOffsetEnd=647 PartOfSpeech=CC] [Text=even CharacterOffsetBegin=648 CharacterOffsetEnd=652 PartOfSpeech=RB] [Text=`` CharacterOffsetBegin=653 CharacterOffsetEnd=654 PartOfSpeech=``] [Text=C.H.O.M.P.S. CharacterOffsetBegin=654 CharacterOffsetEnd=666 PartOfSpeech=NNP] [Text='' CharacterOffsetBegin=671 CharacterOffsetEnd=672 PartOfSpeech=''] [Text=... CharacterOffsetBegin=672 CharacterOffsetEnd=675 PartOfSpeech=:] [Text=okay CharacterOffsetBegin=675 CharacterOffsetEnd=679 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=679 CharacterOffsetEnd=680 PartOfSpeech=,] [Text=maybe CharacterOffsetBegin=681 CharacterOffsetEnd=686 PartOfSpeech=RB] [Text=not CharacterOffsetBegin=687 CharacterOffsetEnd=690 PartOfSpeech=RB] [Text=`` CharacterOffsetBegin=691 CharacterOffsetEnd=692 PartOfSpeech=``] [Text=C.H.O.M.P.S. CharacterOffsetBegin=692 CharacterOffsetEnd=704 PartOfSpeech=NNP] [Text=`` CharacterOffsetBegin=709 CharacterOffsetEnd=710 PartOfSpeech=``] [Text=If CharacterOffsetBegin=710 CharacterOffsetEnd=712 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=713 CharacterOffsetEnd=716 PartOfSpeech=PRP] [Text='ve CharacterOffsetBegin=716 CharacterOffsetEnd=719 PartOfSpeech=VBP] [Text=read CharacterOffsetBegin=720 CharacterOffsetEnd=724 PartOfSpeech=VBN] [Text=the CharacterOffsetBegin=725 CharacterOffsetEnd=728 PartOfSpeech=DT] [Text=book CharacterOffsetBegin=729 CharacterOffsetEnd=733 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=733 CharacterOffsetEnd=734 PartOfSpeech=,] [Text=avoid CharacterOffsetBegin=735 CharacterOffsetEnd=740 PartOfSpeech=VB] [Text=this CharacterOffsetBegin=741 CharacterOffsetEnd=745 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=746 CharacterOffsetEnd=751 PartOfSpeech=NN] [Text=at CharacterOffsetBegin=752 CharacterOffsetEnd=754 PartOfSpeech=IN] [Text=all CharacterOffsetBegin=755 CharacterOffsetEnd=758 PartOfSpeech=DT] [Text=costs CharacterOffsetBegin=759 CharacterOffsetEnd=764 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=764 CharacterOffsetEnd=765 PartOfSpeech=.] 
root(ROOT-0, gives-4)
advmod(dog-3, Only-1)
det(dog-3, the-2)
nsubj(gives-4, dog-3)
det(performance-7, a-5)
amod(performance-7, respectable-6)
dobj(gives-4, performance-7)
punct(gives-4, ,-8)
cc(gives-4, and-9)
mark(want-12, if-10)
nsubj(want-12, you-11)
advcl(better-25, want-12)
aux(see-14, to-13)
xcomp(want-12, see-14)
det(movie-17, a-15)
amod(movie-17, decent-16)
dobj(see-14, movie-17)
prep(movie-17, about-18)
det(dog-20, a-19)
pobj(about-18, dog-20)
punct(better-25, ,-21)
nsubj(better-25, you-22)
aux(better-25, 'd-23)
cop(better-25, be-24)
conj(gives-4, better-25)
prep(better-25, of-26)
pcomp(of-26, watching-27)
punct(watching-27, ``-28)
det(Journey-31, The-29)
amod(Journey-31, Incredible-30)
dobj(watching-27, Journey-31)
punct(watching-27, ''-32)
punct(gives-4, ,-33)
punct(gives-4, ``-34)
dobj(gives-4, Cujo-35)
punct(Cujo-35, ''-36)
cc(Cujo-35, or-37)
advmod(C.H.O.M.P.S.-40, even-38)
punct(C.H.O.M.P.S.-40, ``-39)
conj(Cujo-35, C.H.O.M.P.S.-40)
punct(Cujo-35, ''-41)
punct(Cujo-35, ...-42)
dep(Cujo-35, okay-43)
punct(okay-43, ,-44)
advmod(avoid-57, maybe-45)
neg(C.H.O.M.P.S.-48, not-46)
punct(C.H.O.M.P.S.-48, ``-47)
nsubj(avoid-57, C.H.O.M.P.S.-48)
punct(avoid-57, ``-49)
mark(read-53, If-50)
nsubj(read-53, you-51)
aux(read-53, 've-52)
advcl(avoid-57, read-53)
det(book-55, the-54)
dobj(read-53, book-55)
punct(avoid-57, ,-56)
conj(okay-43, avoid-57)
det(movie-59, this-58)
dobj(avoid-57, movie-59)
prep(avoid-57, at-60)
det(costs-62, all-61)
pobj(at-60, costs-62)
punct(gives-4, .-63)

Sentence #5 (15 tokens):
If you haven't read the book, read it and avoid this movie.
[Text=If CharacterOffsetBegin=770 CharacterOffsetEnd=772 PartOfSpeech=IN] [Text=you CharacterOffsetBegin=773 CharacterOffsetEnd=776 PartOfSpeech=PRP] [Text=have CharacterOffsetBegin=777 CharacterOffsetEnd=781 PartOfSpeech=VBP] [Text=n't CharacterOffsetBegin=781 CharacterOffsetEnd=784 PartOfSpeech=RB] [Text=read CharacterOffsetBegin=785 CharacterOffsetEnd=789 PartOfSpeech=VB] [Text=the CharacterOffsetBegin=790 CharacterOffsetEnd=793 PartOfSpeech=DT] [Text=book CharacterOffsetBegin=794 CharacterOffsetEnd=798 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=798 CharacterOffsetEnd=799 PartOfSpeech=,] [Text=read CharacterOffsetBegin=800 CharacterOffsetEnd=804 PartOfSpeech=VBD] [Text=it CharacterOffsetBegin=805 CharacterOffsetEnd=807 PartOfSpeech=PRP] [Text=and CharacterOffsetBegin=808 CharacterOffsetEnd=811 PartOfSpeech=CC] [Text=avoid CharacterOffsetBegin=812 CharacterOffsetEnd=817 PartOfSpeech=VB] [Text=this CharacterOffsetBegin=818 CharacterOffsetEnd=822 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=823 CharacterOffsetEnd=828 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=828 CharacterOffsetEnd=829 PartOfSpeech=.] 
root(ROOT-0, read-9)
mark(read-5, If-1)
nsubj(read-5, you-2)
aux(read-5, have-3)
neg(read-5, n't-4)
advcl(read-9, read-5)
det(book-7, the-6)
dobj(read-5, book-7)
punct(read-9, ,-8)
dobj(read-9, it-10)
cc(read-9, and-11)
conj(read-9, avoid-12)
det(movie-14, this-13)
dobj(avoid-12, movie-14)
punct(read-9, .-15)

Sentence #6 (22 tokens):
You'll thank me later.A somewhat better translation of a Dean Koontz book is the capable thriller, "Phantoms".
[Text=You CharacterOffsetBegin=834 CharacterOffsetEnd=837 PartOfSpeech=PRP] [Text='ll CharacterOffsetBegin=837 CharacterOffsetEnd=840 PartOfSpeech=MD] [Text=thank CharacterOffsetBegin=841 CharacterOffsetEnd=846 PartOfSpeech=VB] [Text=me CharacterOffsetBegin=847 CharacterOffsetEnd=849 PartOfSpeech=PRP] [Text=later.A CharacterOffsetBegin=850 CharacterOffsetEnd=857 PartOfSpeech=JJ] [Text=somewhat CharacterOffsetBegin=858 CharacterOffsetEnd=866 PartOfSpeech=RB] [Text=better CharacterOffsetBegin=867 CharacterOffsetEnd=873 PartOfSpeech=JJR] [Text=translation CharacterOffsetBegin=874 CharacterOffsetEnd=885 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=886 CharacterOffsetEnd=888 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=889 CharacterOffsetEnd=890 PartOfSpeech=DT] [Text=Dean CharacterOffsetBegin=891 CharacterOffsetEnd=895 PartOfSpeech=NNP] [Text=Koontz CharacterOffsetBegin=896 CharacterOffsetEnd=902 PartOfSpeech=NNP] [Text=book CharacterOffsetBegin=903 CharacterOffsetEnd=907 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=908 CharacterOffsetEnd=910 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=911 CharacterOffsetEnd=914 PartOfSpeech=DT] [Text=capable CharacterOffsetBegin=915 CharacterOffsetEnd=922 PartOfSpeech=JJ] [Text=thriller CharacterOffsetBegin=923 CharacterOffsetEnd=931 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=931 CharacterOffsetEnd=932 PartOfSpeech=,] [Text=`` CharacterOffsetBegin=933 CharacterOffsetEnd=934 PartOfSpeech=``] [Text=Phantoms CharacterOffsetBegin=934 CharacterOffsetEnd=942 PartOfSpeech=NNP] [Text='' CharacterOffsetBegin=942 CharacterOffsetEnd=943 PartOfSpeech=''] [Text=. CharacterOffsetBegin=943 CharacterOffsetEnd=944 PartOfSpeech=.] 
root(ROOT-0, thriller-17)
nsubj(thank-3, You-1)
aux(thank-3, 'll-2)
csubj(thriller-17, thank-3)
dep(translation-8, me-4)
amod(translation-8, later.A-5)
advmod(better-7, somewhat-6)
amod(translation-8, better-7)
dobj(thank-3, translation-8)
prep(translation-8, of-9)
det(book-13, a-10)
nn(book-13, Dean-11)
nn(book-13, Koontz-12)
pobj(of-9, book-13)
cop(thriller-17, is-14)
det(thriller-17, the-15)
amod(thriller-17, capable-16)
punct(thriller-17, ,-18)
punct(thriller-17, ``-19)
dep(thriller-17, Phantoms-20)
punct(Phantoms-20, ''-21)
punct(thriller-17, .-22)

